src/commands/cli/release/build.ts

```diff
@@ -0,0 +1,239 @@
+/*
+ * Copyright (c) 2020, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+import * as os from 'os';
+import { arrayWithDeprecation, Flags, SfCommand, Ux } from '@salesforce/sf-plugins-core';
+import { exec, ExecOptions, set } from 'shelljs';
+import { ensureString } from '@salesforce/ts-types';
+import { Env } from '@salesforce/kit';
+import { Octokit } from '@octokit/core';
+import { bold } from 'chalk';
+import { Messages, SfError } from '@salesforce/core';
+import { PackageRepo } from '../../../repository';
+
+Messages.importMessagesDirectory(__dirname);
+const messages = Messages.loadMessages('@salesforce/plugin-release-management', 'cli.release.build');
+
+export default class build extends SfCommand<void> {
+  public static readonly description = messages.getMessage('description');
+  public static readonly summary = messages.getMessage('description');
+  public static readonly examples = messages.getMessage('examples').split(os.EOL);
+  public static readonly aliases = ['cli:latestrc:build'];
+  public static readonly flags = {
+    'start-from-npm-dist-tag': Flags.string({
+      summary: messages.getMessage('flags.startFromNpmDistTag'),
+      // default: 'latest-rc', // TODO: Will need to update this in GHA before next RC. `exactlyOne` does not work wellwith defaults
+      char: 'd',
+      aliases: ['rctag'],
+      deprecateAliases: true,
+      exactlyOne: ['start-from-npm-dist-tag', 'start-from-github-ref'],
+    }),
+    'start-from-github-ref': Flags.string({
+      summary: messages.getMessage('flags.startFromGithubRef'),
+      char: 'g',
+      exactlyOne: ['start-from-npm-dist-tag', 'start-from-github-ref'],
+    }),
+    'build-only': Flags.boolean({
+      summary: messages.getMessage('flags.buildOnly'),
+      default: false,
+    }),
+    resolutions: Flags.boolean({
+      summary: messages.getMessage('flags.resolutions'),
+      default: true,
+      allowNo: true,
+    }),
+    only: arrayWithDeprecation({
+      summary: messages.getMessage('flags.only'),
+    }),
+    'pinned-deps': Flags.boolean({
+      summary: messages.getMessage('flags.pinnedDeps'),
+      default: true,
+      allowNo: true,
+    }),
+    patch: Flags.boolean({
+      summary: messages.getMessage('flags.patch'),
+      exclusive: ['prerelease'],
+    }),
+    prerelease: Flags.string({
+      summary: messages.getMessage('flags.prerelease'),
+      exclusive: ['patch'],
+    }),
+    snapshot: Flags.boolean({
+      summary: messages.getMessage('flags.snapshot'),
+    }),
+    schema: Flags.boolean({
+      summary: messages.getMessage('flags.schema'),
+    }),
+  };
+
+  public async run(): Promise<void> {
+    // I could not get the shelljs.exec config { fatal: true } to actually throw an error, but this works :shrug:
```

Comment:
I would avoid it and just call a Promisified childprocess.exec.  There's not reason to use shelljs to exec things.